src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs

```diff
@@ -67,7 +67,7 @@ public SyntaxNode Mutate(SyntaxNode currentNode)
                 StatementSyntax ast = statement as StatementSyntax;
 
                 // this is a temporary fix for mutating LocalDeclarationStatements because mutating withing these statements breaks the application
-                // TODO: backlog item Ternary assignment statements mutaties
+                // TODO: backlog item Ternary assignment statements mutates
```

Comment:
the word you corrected means mutations in dutch so I'm pretty sure this should be mutations and not mutates

Reply:
Thanks! I'm not sure about this change, push additional commit.